eth_defi/one_delta/price.py

```diff
@@ -0,0 +1,235 @@
+"""1delta price calculations.
```

Comment:
Can you update docs as well (unfortunately all modules need to be manually added for now)